src/Interactions/DNAInteraction.cpp

```diff
@@ -234,8 +233,10 @@ void DNAInteraction::get_settings(input_file &inp) {
 
 	if(getInputBool(&inp, "use_average_seq", &_average, 0) == KEY_FOUND) {
 		if(!_average) {
+			std::string inter_type;
+			getInputString(&inp, "interaction_type", inter_type, 0);
 
-			if(_drh_interaction){
```

Comment:
@eryykr I have removed the _drh_interaction member altogether, in favour of a local variable that is initialised from the input file. Please have a look and let me know if this makes sense

Reply:
Sure, looks good